src/odemis/util/test/img_test.py

```diff
@@ -451,6 +451,11 @@ def test_irange(self):
 
     def test_fast(self):
         """Test the fast conversion"""
+        try:
+            import odemis.util.img_fast
+        except ImportError as ex:
+            self.skipTest(f"img_fast not available ({ex}), cannot test it")
+
         data = numpy.ones((251, 200), dtype="uint16")
```

Comment:
test_speed is also intermittently failing on 20.04, is this related? 

```
======================================================================
FAIL: test_speed (src.odemis.util.test.img_test.TestFindOptimalRange)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/odemis/odemis/src/odemis/util/test/img_test.py", line 124, in test_speed
    self.assertLessEqual(dur_sc, dur_full)
AssertionError: 0.23967909812927246 not less than or equal to 0.20863556861877441

----------------------------------------------------------------------

```

Reply:
That's annoying... all these speed related tests are quite unreliable (especially in the "cloud"). That issue is not directly related as the "fast" and "slow" paths it tests are both written in numpy. So let's keep it for another time...